app/screens/plugin/index.tsx

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react';
+import {shallowEqual, useSelector} from 'react-redux';
+
+import type {GlobalState} from '@mm-redux/types/store';
+
+import {getTheme} from '@mm-redux/selectors/entities/preferences';
+
+import {SafeAreaView} from 'react-native-safe-area-context';
+import {WebView} from 'react-native-webview';
+
+import type {Theme} from '@mm-redux/types/preferences';
+
+import StatusBar from 'app/components/status_bar';
+import {changeOpacity, makeStyleSheetFromTheme} from 'app/utils/theme';
+
+const HEADERS = {
+    'X-Mobile-App': 'mattermost',
+};
+
+interface PluginProps {
+    link: string;
+}
+
+function Plugin({link}: PluginProps) {
+    const [theme] = useSelector((state: GlobalState) => ([
+        getTheme(state),
+    ]), shallowEqual);
+
+    const style = getStyleSheet(theme);
+
+    const renderWebView = () => {
+        const userAgent = 'Mozilla/5.0 (Linux; Android 10; Android SDK built for x86 Build/LMY48X) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/81.0.4044.117 Mobile Safari/608.2.11';
```

Comment:
Why this user agent?

Reply:
Most of this code is adapted from SSO. This is the user agent user with Google SSO, so I guess that this is to deal with the problem of Google not letting you connect from applications, or something similar (there was this problem with the Desktop app). Since the client has no knowledge on whether the plugin want to go to google or not, we always set it.